### PR TITLE
testing/gnome-latex: new aport

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

File: testing/gnome-latex/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+pkgdesc="LaTeX editor for the GNOME desktop"
+url="https://wiki.gnome.org/Apps/GNOME-LaTeX"
+arch="all"
+license="GPL-3.0-or-later"
+makedepends="glib-dev intltool itstool vala yelp gobject-introspection-dev
+	gsettings-desktop-schemas-dev gspell-dev gtk+3.0-dev libgee-dev
+	gtksourceview4-dev tepl-dev gspell-dev dconf-dev"
+subpackages="$pkgname-doc $pkgname-lang"
+source="https://download.gnome.org/sources/gnome-latex/${pkgver%.*}/gnome-latex-${pkgver}.tar.xz"
+builddir="$srcdir/gnome-latex-$pkgver"
```
Comment (maxice8) [inline testing/gnome-latex/APKBUILD]: idn't this the default value ?

File: testing/gspell/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+pkgdesc="Flexible API to add spell checking to a GTK+ application"
+url="https://wiki.gnome.org/Projects/gspell"
+arch="all"
+license="LGPL-2.1-or-later"
+makedepends="glib-dev vala iso-codes-dev gtk+3.0-dev enchant2-dev vala
+	gobject-introspection-dev"
+subpackages="$pkgname-dev $pkgname-doc $pkgname-lang"
+options="!check" # Needs DISPLAY
+source="https://download.gnome.org/sources/gspell/${pkgver%.*}/gspell-${pkgver}.tar.xz"
+builddir="$srcdir/gspell-$pkgver"
```
Comment (maxice8) [inline testing/gspell/APKBUILD]: default value

**Comment by Ikke** [comment]
On x86: 

> The following tests FAILED:
> 44 - th:tis-620 (Failed)
> Errors while running CTest

**Comment by Cogitri** [comment]
Yup, only happens on drone though 🤔

File: testing/enchant2/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	make
+}
+
+check() {
+	make check
+}
+
+package() {
+	make DESTDIR="$pkgdir" install
+	rm "$pkgdir"/usr/share/enchant/enchant.ordering
```
Comment (Cogitri) [inline testing/enchant2/APKBUILD]: enchant and enchant2 both provide this file, will have to look into how I can share this between them.

Comment (Cogitri) [reply testing/enchant2/APKBUILD]: Fixed.

**Pull request closed without merging**